### PR TITLE
docs(ddl): :memo: describe each DDL test function

### LINKED ISSUE #4: DDL tasks are not described.
In DDL's doc, there are not any description for each step of test case on [Medical appointment](https://github.com/CleverseAcademy/sql-exercise/tree/main/docs/ddl#medical-appointment) and [Organization](https://github.com/CleverseAcademy/sql-exercise/tree/main/docs/ddl#organization).

The tests cannot be passed without information.

So pls update the task description such as initial table schema and what to do with ALTER to [DDL](https://github.com/CleverseAcademy/sql-exercise/tree/main/docs/ddl)

### PR DESCRIPTION
**Pull request by intaniger**
Fix #4

### CODE REVIEW

File: src/ddl/appointment/doctor.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import ddl from "../../engines/ddl";
 
+// Create table with all necessary columns and `department` of 64-character varying type
```
Comment (bunnanna) [inline src/ddl/appointment/doctor.ts:3]: No need to say about row exception.

File: src/ddl/organization/applicant.ts [hunk 2]
```diff
@@ -1,7 +1,10 @@
 import ddl from "../../engines/ddl";
 
+// Create applicant table with all necessary columns and `phone` of INT type and cv of type text
```
Comment (bunnanna) [inline src/ddl/organization/applicant.ts:3]: No need to say about row exception.

File: src/ddl/organization/job_availability.ts [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import ddl from "../../engines/ddl";
 
-export const createJobAvailabilityStmt = ddl(`
-CREATE `);
+// Create applicant table with all necessary columns and `duration` of type INT  and `cost_per_hour` of type INT
```
Comment (bunnanna) [inline src/ddl/organization/job_availability.ts:3]: No need to say about row exception.

**Review by bunnanna — CHANGES_REQUESTED** [review]
The tables' schema and detail are needed when CREATE and after mutations. 
Then add all detail to doc.

File: src/ddl/appointment/availability.ts [hunk 4]
```diff
@@ -2,10 +2,14 @@ import ddl from "../../engines/ddl";
 
 export const createTableStmt = ddl(`CREATE `);
```
[the remark this answers is not in the document]
Comment (intaniger) [reply src/ddl/appointment/availability.ts:3]: added to https://github.com/CleverseAcademy/sql-exercise/tree/fix/ddl-doc/docs/ddl#tables-definition-1

**Pull request merged**